### PR TITLE
DarkTurquoise to ForestGreen 

### LINKED ISSUE #62: Replace DarkTurquoise with ForestGreen across all site
## Problem
Since we defined to start with AA accessibility level, the `DarkTurquoise` color does not match at all the contrast ratio for small and large text sizes. This is a brand concern that we have been discussing and we will come with the final brand solution after designing the .org site.

In parallel, I have been trying to adjust the `DarkTurquoise`, but its nature is low-contrast and bright. Adjusting this color means changing it completely.

I assign myself a task (#48) to work on the interaction color for the `DarkTurquoise`. But as I said above, the issue is bigger than the use on this site.

## Solution
We can replace the `DarkTurquoise` with a different brand color. And based on the AA level and contrast ratio required, the `ForestGreen` seems to match better. For `ForestGreen`, we do have an interaction color called `BrighterForestGreen`. So all `hover` states can use it without problem.

I duplicated the home page replacing the `DarkTurquoise` with `ForestGreen`. So please @krysal @kgodey @TimidRobot comment this ticket to see your thoughts about this change. The accessibility is a crucial dimension of Vocabulary and, unfortunately, the conversation about adjusting brand colors came after starting this project.

### Mockup sample
Screen Shot 2020-08-13 at 09 36 56

### PR DESCRIPTION
**Pull request by Aakash2408**
## Fixes

Fixes #62  by @panchovm

## Description

Replaced DarkTurquoise with ForestGreen across all site

## Technical details
  ForestGreen:RGB(4,166,53)

## Screenshots
Screenshot from 2020-08-19 00-00-21

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Aakash2408** [comment]
I guess there are again issues.I reforked the repository after deleting the old one and still facing this issue

**Comment by kgodey** [comment]
@Aakash2408 you're fine, I updated the branch again. You need to select the correct branch when making the PR in GitHub.

**Comment by kgodey** [comment]
It doesn't look like the changes showed up, though.

**Comment by Aakash2408** [comment]
@kgodey whee are my changes which i just pus

> It doesn't look like the changes showed up, though.

I will just check in local

**Comment by Aakash2408** [comment]
There are done in my local repo
Screenshot from 2020-08-19 00-22-49

**Comment by Aakash2408** [comment]
Why the test is failing

File: legal_db/static/styles.scss [hunk 1]
```diff
@@ -207,7 +207,7 @@ a, a:hover {
 }
 
 footer a:hover {
-  color: $color-dark-turquoise;
+  color: $color-forest-green;
```
Comment (krysal) [inline legal_db/static/styles.scss:210]: This must keep the turquoise color.

Comment (Aakash2408) [reply legal_db/static/styles.scss:210]: Ok i will look into it

Comment (Aakash2408) [reply legal_db/static/styles.scss:210]: > This must keep the turquoise color.

do I  only need to change it back to dark-turquoise on the  hover on the footer

**Comment by Aakash2408** [comment]
> > Why the test is failing @kgodey :(
> 
> I'm not 100%, but I think the test is failing due to secrets not being available to forks (a known issue with PRs from forks: [actions/checkout#298 (comment)](https://github.com/actions/checkout/issues/298#issuecomment-664976337))

Yes I guess the secret_key is not set.

**Comment by Aakash2408** [comment]
@krysal  @TimidRobot  Can you kindly review the changes in code, am not sure about this test?

**Pull request merged**